internal/db/storage/blob/s3/session/option.go

```diff
@@ -90,83 +96,95 @@ func WithToken(tk string) Option {
 	}
 }
 
+// WithMaxRetries returns the option to set the maxRetries.
```

Comment:
Does no problem occurs when r is 0?

Reply:
thank you for comment
I read the below document about s3.
https://github.com/aws/aws-sdk-go/blob/0d8d5fad7f70e9fa8d374081d5c0f91b42f70bea/aws/config.go#L89
and retry logic

In this document, config can be set by allowing 0 and minus. So I don't think there will be any problems. 
And Also, if a minus is set, it will not be retried, so I don't think there is any particular problem.

https://github.com/aws/aws-sdk-go/blob/d1930f00fd9f6a4312fb7a756130ae6e1ec7b4d7/aws/request/request.go#L251